src/packagedcode/spec.py

```diff
@@ -0,0 +1,153 @@
+# All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+import csv
+import glob
+import io
+import logging
+import os
+import re
+
+from gemfileparser import GemfileParser
+
+"""
+Handle Cocoapods(.podspec) and Ruby(.gemspec) files.
+"""
+
+TRACE = False
+
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    import sys
+    logging.basicConfig(stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+
+def parse_spec(location):
+    """
+    Return dictionary contains podspec or gemspec file data.
+    """
+    with io.open(location, encoding='utf-8', closefd=True) as data:
+        lines = data.readlines()
+
+    spec_data = {}
+
+    for line in lines:
+        if '.name' in line:
+            name = line.rpartition('=')
+            spec_data['name'] = get_stripped_data(name[2])
```

Comment:
The benefit of partition is that you have a stable number of items returned so you could write that instead and avoid using a magic index for 2:
```
            _key, _sep, value = line.rpartition('=')
            spec_data['name'] = get_stripped_data(value) 
```

Reply:
Thanks, I will change my code according to this :)